submitit/helpers.py

```diff
@@ -216,3 +217,67 @@ def __enter__(self) -> None:
 
     def __exit__(self, *args):
         os.chdir(self.original_dir)
+
+
+def monitor_jobs(jobs: tp.Sequence[core.Job[core.R]], sleep_time_s: float = 30, test_mode: bool = False):
+    """Continuously monitors given jobs until they are all done or failed.
+
+    Parameters
+    ----------
+    jobs: List[Jobs]
+        A list of jobs to monitor
+    sleep_time_s: int
+        The time (in seconds) between two refreshes of the monitoring.
+        Can't be inferior to 30s.
+    test_mode: bool
+        If in test mode, we do not check the length of sleep_time_s
+
+    Yields
+    ------
```

Comment:
This function is not a generator, it does not yield, it returns ;)
```suggestion
    Returns
    --------
```